.github/workflows/release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Release Twisted
+
+on:
+  release:
```

Comment:
I couldn't actually test this, because I would need to trigger a Release event via
GitHub.  But we can play with this and tune this when we do the next Twisted release.

Reply:
You can create a test release and then delete it.

When creating the release, you can use an existing tag, to prevent creating a new tag inside the repo.

Reply:
What tag can I use?  Can I use a tag that does not have this `release.yaml` file?

Reply:
Another idea, is to have this workflow triggered by each PR.
In this way we have Continuous Integration and we make sure that a PR is not breaking the release script.

The workflow can have a precondition for the final publish step, to skip the publish if a new tag was not created.

Reply:
I see that the workflow already has this condition for the publish...so is best to enable it on any PR and not only on release.

Reply:
> What tag can I use? Can I use a tag that does not have this release.yaml file?

Good question. I think the answer is now.

So I think that you can create a 20.4.0.dev0 tag.

I am not sure what is the best way to mark a version as DEV. that is non-production and not even pre-release.

 I would say that you can try with `.dev0` as this is PEP440... it this doesn't work, try 20.4.0-dev0 based on semver schema.

Thanks!

Reply:
https://twisted.readthedocs.io/en/latest/core/development/policy/release-process.html#version-numbers says that the release numbers need to be time based, so wouldn't that mean `20.10.0.dev0` ?

According to https://twisted.readthedocs.io/en/latest/core/development/policy/release-process.html#announce , the `dev0` postfix is added by incremental.